esmvaltool/diag_scripts/weighting/climwip/main.py

```diff
@@ -275,6 +271,12 @@ def main(cfg):
             independence, independence_contributions)
         visualize_and_save_independence(overall_independence, cfg,
                                         model_ancestors)
+        if independence_sigma is None:
+            errmsg = ' '.join([
+                'independence_sigma must be set if independence_contributions',
+                'is set'
+            ])
+            raise NotImplementedError(errmsg)
```

Comment:
This will also work 😅

```suggestion
        if independence_sigma is None:
            raise NotImplementedError('`independence_sigma` must be set if '
                                      '`independence_contributions` is set')
```

Reply:
right! i need to get used to this :) learned to do it always with join